src/components/SidePanel/SidePanel.tsx

```diff
@@ -134,6 +141,12 @@ class SidePanel extends React.Component<ISidePanelProps, ISidePanelState, {}> {
 			Sets the initial width in pixels. The actual width may change if the user
 			resizes it.
 		`,
+		minWidth: number`
+		Sets the minimum width of the Side Panel.
+		`,
+		maxWidth: number`
```

Comment:
As I'm looking at this I'm realizing what would be _really_ nice if we could prevent the panel from going off the left side of the screen. Would it be possible to support something like `100%`? I haven't dug into this component so I don't even know if it's technically feasible.